dot/simswap/util/reverse2original.py

```diff
@@ -131,7 +125,9 @@ def reverse2wholeimage(
         mat_rev[1][1] = -mat[0][0] / div2
         mat_rev[1][2] = -(mat[0][2] * mat[1][0] - mat[0][0] * mat[1][2]) / div2
 
-        orisize = (oriimg.shape[1], oriimg.shape[0])
+        mat_rev = torch.tensor(mat_rev)[None, ...].float()
+
```

Comment:
can you add a comment explaining why we do this below?
```
if tgt_mask.sum() >= 5000:
```

Reply:
This was implemented by the original authors. They didn't add a comment explaining it. I think it means if the mask is big enough.

Reply:
Comment added https://github.com/sensity-ai/dot/blob/feature/3-speedup-simswap/dot/simswap/util/reverse2original.py#L150